src/data/ar-io-data-source.test.ts

```diff
@@ -64,7 +64,7 @@ beforeEach(async () => {
 
   dataSource = new ArIODataSource({
     log,
-    arIO: ArIO.init(),
+    arIO: ArIO.init() as any,
```

Comment:
I'm seeing several new TypeScript errors in these test files when building the images for test containers and with `tsc`

```
> yarn tsc --noEmit
yarn run v1.22.19
$ /Users/derek/ar-io/ar-io-node/node_modules/.bin/tsc --noEmit
src/data/ar-io-data-source.test.ts:67:5 - error TS2741: Property 'getPrescribedNames' is missing in type 'ArIOReadable' but required in type 'AoIORead'.

67     arIO: ArIO.init(),
       ~~~~

  node_modules/@ar.io/sdk/lib/types/io.d.ts:75:5
    75     getPrescribedNames(epoch?: EpochInput): Promise<string[]>;
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    'getPrescribedNames' is declared here.
  src/data/ar-io-data-source.ts:58:5
    58     arIO: AoIORead;
           ~~~~
    The expected type comes from property 'arIO' which is declared here on type '{ log: Logger; arIO: AoIORead; nodeWallet?: string | undefined; maxHopsAllowed?: number | undefined; requestTimeoutMs?: number | undefined; updatePeersRefreshIntervalMs?: number | undefined; }'

src/data/ar-io-data-source.test.ts:206:13 - error TS6133: 'receivedData' is declared but its value is never read.

206         let receivedData = '';
                ~~~~~~~~~~~~

src/data/gateway-data-source.test.ts:163:13 - error TS6133: 'receivedData' is declared but its value is never read.

163         let receivedData = '';
                ~~~~~~~~~~~~

src/data/read-through-data-cache.test.ts:288:11 - error TS6133: 'calledWithArgument' is declared but its value is never read.

288       let calledWithArgument: string;
              ~~~~~~~~~~~~~~~~~~

src/data/read-through-data-cache.test.ts:310:13 - error TS6133: 'receivedData' is declared but its value is never read.

310         let receivedData = '';
                ~~~~~~~~~~~~

src/data/read-through-data-cache.test.ts:428:13 - error TS6133: 'receivedData' is declared but its value is never read.

428         let receivedData = '';
                ~~~~~~~~~~~~

src/data/tx-chunks-data-source.test.ts:20:1 - error TS6133: 'Readable' is declared but its value is never read.

20 import { Readable } from 'node:stream';
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

src/data/tx-chunks-data-source.test.ts:138:28 - error TS6133: 'chunk' is declared but its value is never read.

138           for await (const chunk of data.stream) {
                               ~~~~~

src/data/tx-chunks-data-source.test.ts:168:30 - error TS6133: 'chunk' is declared but its value is never read.

168             for await (const chunk of data.stream) {
                                 ~~~~~

src/routes/graphql/resolvers.test.ts:72:42 - error TS2345: Argument of type '{ id: string; anchor: string; signature: string; recipient: string; ownerAddress: string; ownerKey: string; fee: string; quantity: string; dataSize: string; contentType: undefined; blockIndepHash: string; blockTimestamp: number; height: number; blockPreviousBlock: string; parentId: null; }' is not assignable to parameter of type 'GqlTransaction'.
  Type '{ id: string; anchor: string; signature: string; recipient: string; ownerAddress: string; ownerKey: string; fee: string; quantity: string; dataSize: string; contentType: undefined; blockIndepHash: string; blockTimestamp: number; height: number; blockPreviousBlock: string; parentId: null; }' is missing the following properties from type 'GqlTransaction': blockTransactionIndex, dataItemId, tags, indexedAt

72     const recipient = resolveTxRecipient(GQL_TX);
                                            ~~~~~~

src/routes/graphql/resolvers.test.ts:79:42 - error TS2345: Argument of type '{ recipient: undefined; id: string; anchor: string; signature: string; ownerAddress: string; ownerKey: string; fee: string; quantity: string; dataSize: string; contentType: undefined; blockIndepHash: string; blockTimestamp: number; height: number; blockPreviousBlock: string; parentId: null; }' is not assignable to parameter of type 'GqlTransaction'.
  Type '{ recipient: undefined; id: string; anchor: string; signature: string; ownerAddress: string; ownerKey: string; fee: string; quantity: string; dataSize: string; contentType: undefined; blockIndepHash: string; blockTimestamp: number; height: number; blockPreviousBlock: string; parentId: null; }' is missing the following properties from type 'GqlTransaction': blockTransactionIndex, dataItemId, tags, indexedAt

79     const recipient = resolveTxRecipient(tx);
                                            ~~

src/routes/graphql/resolvers.test.ts:88:38 - error TS2345: Argument of type '{ contentType: string; id: string; anchor: string; signature: string; recipient: string; ownerAddress: string; ownerKey: string; fee: string; quantity: string; dataSize: string; blockIndepHash: string; blockTimestamp: number; height: number; blockPreviousBlock: string; parentId: null; }' is not assignable to parameter of type 'GqlTransaction'.
  Type '{ contentType: string; id: string; anchor: string; signature: string; recipient: string; ownerAddress: string; ownerKey: string; fee: string; quantity: string; dataSize: string; blockIndepHash: string; blockTimestamp: number; height: number; blockPreviousBlock: string; parentId: null; }' is missing the following properties from type 'GqlTransaction': blockTransactionIndex, dataItemId, tags, indexedAt

88     const dataResult = resolveTxData(tx);
                                        ~~

src/routes/graphql/resolvers.test.ts:95:40 - error TS2345: Argument of type '{ id: string; anchor: string; signature: string; recipient: string; ownerAddress: string; ownerKey: string; fee: string; quantity: string; dataSize: string; contentType: undefined; blockIndepHash: string; blockTimestamp: number; height: number; blockPreviousBlock: string; parentId: null; }' is not assignable to parameter of type 'GqlTransaction'.

95     const quantity = resolveTxQuantity(GQL_TX);
                                          ~~~~~~

src/routes/graphql/resolvers.test.ts:102:30 - error TS2345: Argument of type '{ id: string; anchor: string; signature: string; recipient: string; ownerAddress: string; ownerKey: string; fee: string; quantity: string; dataSize: string; contentType: undefined; blockIndepHash: string; blockTimestamp: number; height: number; blockPreviousBlock: string; parentId: null; }' is not assignable to parameter of type 'GqlTransaction'.

102     const fee = resolveTxFee(GQL_TX);
                                 ~~~~~~

src/routes/graphql/resolvers.test.ts:109:34 - error TS2345: Argument of type '{ id: string; anchor: string; signature: string; recipient: string; ownerAddress: string; ownerKey: string; fee: string; quantity: string; dataSize: string; contentType: undefined; blockIndepHash: string; blockTimestamp: number; height: number; blockPreviousBlock: string; parentId: null; }' is not assignable to parameter of type 'GqlTransaction'.

109     const owner = resolveTxOwner(GQL_TX);
                                     ~~~~~~


Found 15 errors in 5 files.

Errors  Files
     2  src/data/ar-io-data-source.test.ts:67
     1  src/data/gateway-data-source.test.ts:163
     3  src/data/read-through-data-cache.test.ts:288
     3  src/data/tx-chunks-data-source.test.ts:20
     6  src/routes/graphql/resolvers.test.ts:72
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```